presto-base-jdbc/src/test/java/io/prestosql/plugin/jdbc/jmx/TestJmxStats.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.jdbc.jmx;
+
+import com.google.common.collect.ImmutableMap;
+import io.prestosql.plugin.jdbc.JdbcPlugin;
+import io.prestosql.plugin.jdbc.TestingH2JdbcModule;
+import io.prestosql.spi.Plugin;
+import io.prestosql.spi.connector.ConnectorFactory;
+import io.prestosql.testing.TestingConnectorContext;
+import org.testng.annotations.Test;
+
+import javax.management.MBeanInfo;
+import javax.management.MBeanServer;
+import javax.management.ObjectName;
+
+import static com.google.common.collect.Iterables.getOnlyElement;
+import static java.lang.String.format;
+import static java.lang.management.ManagementFactory.getPlatformMBeanServer;
+import static org.assertj.core.util.Arrays.isNullOrEmpty;
+import static org.testng.Assert.assertFalse;
+
+public class TestJmxStats
+{
+    @Test
+    public void testJmxStatsExposure()
+            throws Exception
+    {
+        Plugin plugin = new JdbcPlugin("base-jdbc", new TestingH2JdbcModule());
+        ConnectorFactory factory = getOnlyElement(plugin.getConnectorFactories());
+        factory.create("test", ImmutableMap.of("connection-url", "jdbc"), new TestingConnectorContext());
+        MBeanServer mbeanServer = getPlatformMBeanServer();
+        for (ObjectName objectName : mbeanServer.queryNames(new ObjectName("io.prestosql.plugin.jdbc:*"), null)) {
```

Comment:
Please make sure that `mbeanServer.queryNames(new ObjectName("io.prestosql.plugin.jdbc:*")` does not return empty list. Also please verify that few that we care of are there (jdbc client and coonnection factory)